populator/nosql/couchbase-ftest/src/test/java/org/jboss/arquillian/populator/nosql/couchbase/CouchbaseTest.java

```diff
@@ -0,0 +1,41 @@
+package org.jboss.arquillian.populator.nosql.couchbase;
+
+import com.couchbase.client.java.Bucket;
+import com.couchbase.client.java.CouchbaseCluster;
+import com.couchbase.client.java.document.JsonDocument;
+import com.couchbase.client.java.document.json.JsonObject;
+import org.arquillian.cube.HostIp;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.populator.nosql.api.NoSqlPopulator;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(Arquillian.class)
+public class CouchbaseTest {
+
+    @HostIp
+    private String hostIp;
+
+    @ArquillianResource
+    @Couchbase
+    NoSqlPopulator populator;
+
+    @Test
+    public void should_find_books() {
+        populator.forServer(hostIp, 0)
+                .withDatabase("travel-sample")
+                .usingDataSet("/airlines.json")
```

Comment:
One potential improvement here would be to avoid `/` prefix. It's a bit confusing for the user, as he might assume it's in the root OS folder. Also not really needed,